beakerx/setupbase.py

```diff
@@ -323,14 +324,10 @@ def install_kernel(source_kernelspec='', kernelspec_name=None):
                     classpath = (os.path.abspath(os.path.join(target_dir, 'base', 'lib', '*')) + (';' if sys.platform == 'win32' else ':') + os.path.abspath(os.path.join(target_dir, name, 'lib', '*'))).replace('\\', '/')
```

Comment:
this should use json.dumps and OS independent logic the same as install.py does.

Reply:
done.